File: Algorithms/1051. Height Checker/php/solution.php
<?php
class Solution
{

    /**
     * @param Integer[] $heights
     * @return Integer
     */
    function heightChecker($heights)
    {
        $sortedHeights = $heights;
        sort($sortedHeights);
        $count = 0;
        for ($i = 0; $i < count($heights); $i++) {
            if ($heights[$i] != $sortedHeights[$i]) {
                $count++;
            }
        }
        return $count;
    }
}

File: Algorithms/1051. Height Checker/rust/solution.rs
impl Solution {
    pub fn height_checker(heights: Vec<i32>) -> i32 {
        let mut sorted_heights: Vec<i32> = heights.clone();
        sorted_heights.sort();
        let mut count: i32 = 0;
        for i in 0..heights.len() {
            if heights[i] != sorted_heights[i] {
                count += 1;
            }
        }
        count
    }
}
